vs2015/activate.bat

```diff
@@ -73,6 +43,10 @@ set "CFLAGS=%CFLAGS% -MD -GL"
 set "CXXFLAGS=%CXXFLAGS% -MD -GL"
 set "LDFLAGS_SHARED=%LDFLAGS_SHARED% -LTCG ucrt.lib"
 
+:: set CC and CXX for cmake
+set "CXX=cl.exe"
+set "CC=cl.exe"
```

Comment:
@msarahan, need a rebuild for this small change.

Reply:
Maybe rebuild after merging https://github.com/AnacondaRecipes/aggregate/pull/138 ?